packages/react/src/components/Dropdown/DropdownSelectedItem.tsx

```diff
@@ -179,8 +179,8 @@ DropdownSelectedItem.create = createShorthandFactory({
 })
 
 /**
- * A a sub-component of multiple-selection Dropdown.
- * Used to display selected item.
+ * A DropdownSelectedItem represents item of 'multiple-selection' Dropdown.
+ * Used to display the search input field.
```

Comment:
Maybe "to display **IN**  the search input...", as this a selected item which is displayed in the input

Reply:
this is an awesome catch - completely overlooked this moment. Actually, there is nothing in this component to deal with `search` input field - this component just displays selected options, and there is no need to dig deeper in the presentational aspects. Will make the necessary changes here, thank you!